lms/templates/courseware/info.html

```diff
@@ -47,12 +47,10 @@ <h2 class="title">${_("You are not enrolled yet")}</h2>
   <div class="info-wrapper">
     % if user.is_authenticated():
     <section class="updates">
-      % if staff_access and masquerade is not UNDEFINED and studio_url is not None:
-        % if masquerade == 'staff':
-          <div class="wrap-instructor-info studio-view">
-            <a class="instructor-info-action" href="${studio_url}">${_("View Updates in Studio")}</a>
-          </div>
-        % endif
+      % if studio_url is not None and masquerade and masquerade.role == 'staff':
+        <div class="wrap-instructor-info studio-view">
+          <a class="instructor-info-action" href="${studio_url}">${_("View Updates in Studio")}</a>
```

Comment:
@singingwolfboy Should this string be parameterized to no longer hard code "Studio"? Is the constant available in LMS or only in Studio itself?